group_vars/galaxyservers.yml

```diff
@@ -64,20 +60,21 @@ galaxy_separate_privileges: true
 galaxy_manage_paths: true
 galaxy_layout: root-dir
 galaxy_root: /mnt/galaxy
-galaxy_user:
-  name: galaxy
-  shell: /bin/bash
+#galaxy_user: ## shifted to all.yml
+#  name: galaxy
+#  shell: /bin/bash
 galaxy_commit_id: release_20.05
 galaxy_config_style: yaml
 galaxy_force_checkout: true
+galaxy_handler_count: 2
 
 galaxy_tmp_path: "{{ galaxy_root }}/tmp"
 galaxy_australia_website: "https://usegalaxy-au.github.io"
 
 galaxy_server_dir: "{{ galaxy_root }}/galaxy-app"
 galaxy_mutable_config_dir: "{{ galaxy_root }}/var"
 galaxy_mutable_data_dir: "{{ galaxy_root }}"
-# galaxy_config_dir: "{{ galaxy_server_dir }}/config" # this cannot be set this way as config paths are set before the galaxy repo is cloned: the path cannot already exist
+galaxy_config_dir: "{{ galaxy_root }}/config" # this cannot be set this way as config paths are set before the galaxy repo is cloned: the path cannot already exist #This is normally set as {{ galaxy_root }}/config
```

Comment:
This is a much better arrangement.  The comment on this line does not need to be there anymore